diff -urN prozilla-1.3.7.3.org/src/main.c prozilla-1.3.7.3/src/main.c
--- prozilla-1.3.7.3.org/src/main.c	2005-01-23 12:09:59.000000000 +0100
+++ prozilla-1.3.7.3/src/main.c	2005-03-23 22:25:57.000000000 +0100
@@ -1290,6 +1290,7 @@
     int use_server = -1;
     int same_url = FALSE;
     char *conf_file;
+    char *opt_file = NULL;
     char global_conf_file[] = GLOBAL_CONF_FILE;
     /*
      * set the default runtime configuration 
@@ -1303,7 +1304,7 @@
     debug_init();
 
     while ((c =
-	    getopt_long(argc, argv, "?hrfk:1Lt:vgsP:", long_opts,
+	    getopt_long(argc, argv, "?hrfk:1Lt:vgsP:O:", long_opts,
 			NULL)) != EOF)
     {
 	switch (c)
@@ -1361,7 +1362,9 @@
 	     */
 	    rt.use_netrc = FALSE;
 	    break;
-
+	case 'O':
+	    opt_file = kstrdup(optarg);
+	    break;
 	case 'P':
 	    /*
 	     * Save the downloaded file to DIR 
@@ -1753,6 +1756,9 @@
 	}
     }
 
+    if (opt_file)
+	url_data.file=opt_file;
+	
     if ((url_data.proto != URLHTTP) && (url_data.proto != URLFTP))
     {
 	die("Error: Unsupported Protocol was specified\n");
