--- prozilla-1.3.6/acinclude.m4~	Fri Aug 17 16:38:55 2001
+++ prozilla-1.3.6/acinclude.m4	Sun Aug 19 17:19:39 2001
@@ -376,7 +17,7 @@
 dnl @version $Id$
 dnl @author Caolan McNamara <caolan@skynet.ie>
 dnl
-AC_DEFUN(AC_caolan_FUNC_WHICH_GETHOSTBYNAME_R,
+AC_DEFUN([AC_caolan_FUNC_WHICH_GETHOSTBYNAME_R],
 [AC_CACHE_CHECK(for which type of gethostbyname_r, ac_cv_func_which_gethostname_r, [
 AC_CHECK_FUNC(gethostbyname_r, [
         AC_TRY_COMPILE([
